screens/rewards/RewardsScreen.js

```diff
@@ -73,6 +82,42 @@ export default class RewardsScreen extends React.Component {
       return null;
     }
 
+    if (this.state.isGuest) {
+      return (
+        <View style={{ flex: 1 }}>
+          <NavHeaderContainer vertical backgroundColor={Colors.primaryGreen}>
+            <NavButton onPress={() => this.props.navigation.goBack()}>
+              <FontAwesome5 name="arrow-down" solid size={24} color="white" />
+            </NavButton>
+            <BigTitle
+              style={{
+                marginLeft: 18,
+                color: Colors.lightest,
+                fontSize: 36,
+              }}>
+              How it Works
+            </BigTitle>
+          </NavHeaderContainer>
+          <Image
+            source={require('../../assets/images/guest.jpeg')}
+            style={{
+              width: 343,
+              height: 537,
+              display: 'flex',
+              alignContent: 'center',
```

Comment:
Should be `alignSelf` to center

Reply:
Alternative method to avoid fixed pixel width and height: use `marginLeft` and `marginRight: 'auto'` instead, with a fixed percentage width like 80 or 90% and no specified height.